source/Ship.cpp

```diff
@@ -3614,7 +3614,8 @@ double Ship::MinimumHull() const
 		return absoluteThreshold;
 	
 	double thresholdPercent = attributes.Get("threshold percentage");
-	double minimumHull = maximumHull * (thresholdPercent > 0. ? min(thresholdPercent, 1.) : max(.15, min(.45, 10. / sqrt(maximumHull))));
+	double transition = 1 / (1 + 0.0006 * maximumHull);
+	double minimumHull = maximumHull * (thresholdPercent > 0. ? min(thresholdPercent, 1.) : 0.05 * (1. - transition) + 0.75 * transition);
```

Comment:
I like these numbers better.
```suggestion
	double transition = 1 / (1 + 0.0005 * maximumHull);
	double minimumHull = maximumHull * (thresholdPercent > 0. ? min(thresholdPercent, 1.) : 0.1 * (1. - transition) + 0.5 * transition);
```
A similar impact on ships in the middle with less of an impact for ships on the high end compared to the previous numbers and almost no impact compared to the current equation for ships on the lowest end.

Ships from 0 to 600 hull get disabled at near the same point as the current equation.
Ships from 600 to 14000 hull get disabled sooner. (The peak change is that ships around 4500 hull have a disabled threshold about 50% higher.)
Ships above 14000 hull get disabled later.